tools/boards.txt.py

```diff
@@ -761,6 +762,13 @@
     ####################### menu.resetmethod
 
     'resetmethod_menu': collections.OrderedDict([
+        ( '.menu.ResetMethod.ck', 'ck' ),
+        ( '.menu.ResetMethod.ck.upload.resetmethod', 'ck' ),
+        ( '.menu.ResetMethod.nodemcu', 'nodemcu' ),
+        ( '.menu.ResetMethod.nodemcu.upload.resetmethod', 'nodemcu' ),
+        ]),
+
+    'resetmethod_menu_all': collections.OrderedDict([
```

Comment:
What do you think of calling this one ```_extra``` and leaving in it only ```none``` and ```dtrset```, then use it only in generic boards?